hooks/hooks.py

```diff
@@ -14,20 +13,30 @@
 
 
 hooks = hookenv.Hooks()
-
-CONF_PATH = '/etc/consul.json'
+# The path to the consul configuration file.
+CONF_FILE = path('/etc/consul.json')
+# The path to the consul binary.
 CONSUL = path('/usr/local/bin/consul')
-PORTS = [53, 8302, 8400, 8500]
-LEADER_DATA = {'shared-key': 'shared_key'}
+# The path to the JSON file containing default configuration values.
```

Comment:
Very high density of comments here lowers the readability of the code.  I would recommend renaming the variables to be more expressive of their purpose (if unclear) and utilize docstrings to express general communication about code.